### PR TITLE
Added release notes for MP 4.11.0

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

https://enterprisedb.atlassian.net/browse/DOCS-908
https://enterprisedb.atlassian.net/browse/MIG-5173

### CODE REVIEW

File: product_docs/docs/migration_portal/4/01_mp_release_notes/mp_4.11.0_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+Released: 20 Aug 2024
+
+New features, enhancements, bug fixes, and other changes in Migration Portal 4.11.0 include the following:
+
+| Type              | Description                                                                                                      |
+|-------------------|------------------------------------------------------------------------------------------------------------------|
+| Enhancement       | Updated the styling for logos, fonts and layouts for an improved look and feel.                                  |
+| Security&nbsp;fix | Updated the software libraries and implemented other security improvements.                                      |
```
Comment (mlewandowski-edb) [inline product_docs/docs/migration_portal/4/01_mp_release_notes/mp_4.11.0_rel_notes.mdx:13]: I recommend changing "Updated the software libraries and ..." to "Updated software libraries and ..."

**Review by mlewandowski-edb — CHANGES_REQUESTED** [review]
I reviewed and submitted a comment suggesting a minor edit. Otherwise it looks good.

**Comment by gvasquezvargas** [comment]
@mlewandowski-edb  implemented your suggestion ✅ 
Can you please approve? I will wait for the green flag via mail or ping before merging.

**Review by mlewandowski-edb — APPROVED** [review]

**Review by djw-m — APPROVED** [review]
LGTM
But in future - 
Suggest that Security Fix should really only be used for a specific issue - Customers will ask questions if they read the release notes.
And are there really no related support tickets to any of these issues?
(Happy to talk release notes as assisting on a general effort to standardize the process of creating them).

**Pull request merged**